src/pages/questions.js

```diff
@@ -0,0 +1,29 @@
+import {nanoid} from 'nanoid';
+import {Helmet} from 'react-helmet';
+
+import Layout from '../components/Layout';
+import QuestionComponent from '../components/QuestionComponent';
+
+export default function QuestionPage() {
+	const questions = [
+		{id: nanoid(), questionText: 'What is export default?'},
+		{id: nanoid(), questionText: 'What does clean code means?'},
+		{id: nanoid(), questionText: 'Why do we have to define keys?'},
+	];
+
+	return (
+		<Layout>
+			<Helmet>
+				<title key="title">My Project</title>
+				<meta key="description" name="description" content="This is my project" />
+			</Helmet>
```

Comment:
I would like to suggest that you adjust the title and the content. 🙂

Reply:
Will be done in a sec :)